firebase/functions/index.js

```diff
@@ -27,118 +38,116 @@ exports.resultCounter = functions.database.ref('/v2/results/{projectId}/{groupId
         return null
     }
 
-    // Firebase Realtime Database references
-    const groupRef = admin.database().ref('/v2/groups/' + context.params.projectId + '/' + context.params.groupId)
-    const userRef = admin.database().ref('/v2/users/' + context.params.userId)
-    const userContributionRef = userRef.child('contributions/' + context.params.projectId)
-
-    // Counter for User contributions across Projects
-    const totalGroupContributionCount = userRef.child('groupContributionCount').transaction((currentCount) => {
-        return currentCount + 1
-    });
-    promises.push(totalGroupContributionCount)
-
-    const totalTaskContributionCount = groupRef.child('numberOfTasks').once('value')
-        .then((dataSnapshot) => {
-            const numberOfTasks = dataSnapshot.val()
-            return numberOfTasks
-        })
-        .then((numberOfTasks) => {
-            return userRef.child('taskContributionCount').transaction((currentCount) => {
-                return currentCount + numberOfTasks
-            })
-        })
-    promises.push(totalTaskContributionCount)
-
-    // Counter for User contributions on a distict Project
-    const groupContributionCount = userContributionRef.child('groupContributionCount').transaction((currentCount) => {
```

Comment:
I think we were a bit too hasty in removing this number, it's used in the leaderboard :fearful:

Reply:
okay, so this will then also have implications on the code in general. What's your feeling should these stats by set in realtime (by a firebase function) or updated just once in a while (e.g. every 15 minutes by the backend)?

My feeling is, that since these attributes are not used in the app, it doesn't have to be realtime.

Reply:
okay some further thoughts: We could for now keep it in Firebase and use the functions we already have. When re-designing the user profile section we can have this discussion again and decide which user stats we would like to provide and how they should be calculated.

When re-designing the user profile we can also think about this will affect the leaderboard. Ideally we can bring those two things closer together and maybe we do not need a leaderboard when we improve the user profile.